server.js

```diff
@@ -2,6 +2,11 @@ const express = require('express');
 
 const server = express();
 
+Foobar.get('data/accounts-model.js', (req, res) => {
+    return res.send('access to new database')
+}
+)
+
```

Comment:
Still waiting on those endpoints to be done.

Reply:
end points added